graftm/hmmer.py

```diff
@@ -462,10 +399,16 @@ def extractMultipleHits(reads_path, stats): # TODO: Not used for the time being
             cmd = "%s %s | awk '{print \">\" substr($0,2);getline;print;getline;getline}' > %s" % (fxtract_cmd, raw_sequences_path, output_path)
         else:
             raise Exception("Programming error")
-        
         logging.debug("Running command: %s" % cmd)
         subprocess.check_call(cmd, shell=True)
-
+        
+        # If any of the hits have more than one alignment span (i.e. >1 hit)
+        if any([x for x in hits.values() if len(x) > 1]):
```

Comment:
extractMultipleHits reads in all of the sequences with biopython, which can take time. If all the reads are hit only once, then it is better to skip this process if they are just being re-written to file anyways.